### PR TITLE
Jazzband transfer tasks

### PR DESCRIPTION
**Pull request by Archmonger**
Handles jazzband transfer tasks listed in #417

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[![Jazzband](https://jazzband.co/static/img/jazzband.svg)](https://jazzband.co/)
```
Comment (jezdez) [inline contributing.md:1]: @Archmonger Let's upper-case the filename please.

Comment (Archmonger) [reply contributing.md:1]: https://jazzband.co/static/img/Jazzband.svg and JAZZBAND.svg is are 404. Perhaps I'm misunderstanding?

**Comment by Archmonger** [comment]
@johnthagen I'm not sure how to integrate with jazzband's codecov. Things might still be linked to the [original codecov page](https://coveralls.io/github/django-dbbackup/django-dbbackup?branch=master)?

Do you have any advice here?

**Comment by Archmonger** [comment]
I've added the jazzband bot to RTD. For backup purposes, I'd also like to add @johnthagen but I don't know your RTD username.

**Comment by johnthagen** [comment]
@Archmonger My RTD username is `johnthagen`

**Comment by Archmonger** [comment]
Alright. You've been added to RTD.

**Comment by Archmonger** [comment]
@johnthagen 
I think we should merge this PR.

I don't know what else needs to be done to get codecov disconnected from the old URL and then connected to this project in Jazzband. And also not entirely sure what else needs to be customized with our project release CI.

But these can be handled on separate PRs.

**Review by johnthagen — APPROVED** [review]

**Pull request merged**